data/ShanmukhiKairuppala.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Shanmukhi Kairuppala",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " I'm an Open Source enthusiast and an aesthete!",
+  "avatar": "https://github.com/ShanmukhiKairuppala.png",
+  "links": [
```

Comment:
You need commas between each object in the links array

Reply:
That is correct, I will mention below the places where the comma (,) has to be added!